docs/guides/audio-config.md

```diff
@@ -54,6 +53,13 @@ An example to get PipeWire working on Ubuntu, which uses PulseAudio by default i
 The "Audio Volume" dialog / Audio in System Settings allow users to "Raise maximum volume", allowing to go past 100%. This
 does not offer a great deal of flexibility, it might work for getting acceptable recordings however.
 
+## Speakers
+
+Currently we have an experimental DSP config for 16,1 with 6 speakers.
```

Comment:
16,4 and 16,1 share the same config. So probably add that as well.

Reply:
It might be worth saying "DSP (<what DSP stands for>)", as some people (including me!) don't know what that means.

Reply:
> 16,4 and 16,1 share the same config. So probably add that as well.

first It should be checked if the use same speakers model